### DISCUSSION TITLE
Feature request (mux-player): add chapters, chapter markers

### DISCUSSION THREAD

**Discussion by aminamos** [muxinc/elements#448] [Q&A] [answered]
Provide the ability (data source, or built-in) to integrate chapters with Mux Player.

**Comment by cameroncovey-iew** [comment]
Which Mux Elements/Packages would you like a new feature for?
Mux Player

🎥 Describe the feature you'd like to request
Add ability to set chapter markers and list out the chapters for selecting that timestamp in playback for viewer. Either a popup box that contains a list of chapters, or maybe in line with play bar.

💻 Describe the use cases the feature will solve
Viewers wanting to go to part of a video at a click instead of having to scrub through video.

⭐ Describe the solution you'd like to see
Either a popup box that contains a list of chapters, or maybe in line with play bar. (see how Youtube does it)

🔄 Describe alternatives or workarounds you've considered
We use Bitmovin Player and it has chapters (set up in an XML type document for video). We need the chpaters so we can direct viewers to parts of the video to watch.

**Comment by TheEagleByte** [comment]
Can we prioritize this? It's one of the key features missing that we get with Bitmovin's Player.

**Comment by luwes** [comment, marked as answer]
Chapters support has been added recently to Mux player and media-chrome.
https://docs.mux.com/guides/player-advanced-usage#add-chapters-and-time-based-metadata
